utils/hct/hctdb.py

```diff
@@ -5850,6 +5855,97 @@ def UFI(name, **mappings):
         )
         next_op_idx += 1
 
+        self.add_dxil_op(
+            "MatVecMul",
+            next_op_idx,
+            "MatVecMul",
+            "Multiplies a MxK dimension matrix and a K sized input vector",
+            "<hfwi,<hfwi",
+            "",
```

Comment:
Isn't this operation read-only with respect to memory referenced through the resource?
```suggestion
            "ro",
```